src/ripple/protocol/impl/SField.cpp

```diff
@@ -119,6 +120,7 @@ SF_U32 const sfCancelAfter(access, STI_UINT32, 36, "CancelAfter");
 SF_U32 const sfFinishAfter(access, STI_UINT32, 37, "FinishAfter");
 SF_U32 const sfSignerListID(access, STI_UINT32, 38, "SignerListID");
 SF_U32 const sfSettleDelay(access, STI_UINT32, 39, "SettleDelay");
+SF_U32 const sfNegativeUNLLgrSeq(access, STI_UINT32, 40, "BeginLedgerSeq");
```

Comment:
I'd remove this. Prefer using `sfFirstLedgerSequence` (we mark it as deprecated, but we can remove that).

Reply:
Thanks. Using `sfFirstLedgerSequence` now.